lib/base/pages/home/services/custom_search_location/online_search_location.dart

```diff
@@ -76,7 +76,7 @@ class OnlineSearchLocation implements SearchLocationRepository {
   Future<LocationDetail> reverseGeodecoding(LatLng location) async {
     final response = await http.get(
       Uri.parse(
-        "https://photon.stadtnavi.eu/pelias/v1/reverse?point.lat=${location.latitude}&point.lon=${location.longitude}&boundary.circle.radius=0.1&lang=en&size=1&layers=address&zones=1",
+        "https://photon-eu.stadtnavi.eu/pelias/v1/reverse?point.lat=${location.latitude}&point.lon=${location.longitude}&boundary.circle.radius=0.1&lang=en&size=1&layers=address&zones=1",
```

Comment:
Shouldn't language depend on user's locale?

Reply:
@hbruch  I implemented this in a new PR because I needed to use a branch name that follows the standard.